Views/backend/connect/controller/main.js

```diff
@@ -120,8 +120,33 @@ Ext.define('Shopware.apps.Connect.controller.Main', {
     init: function () {
         var me = this;
 
-        me.initializeParams();
+        if (!window.marketplaceName) {
+            me.sendAjaxRequest(
+                '{url controller=Connect action=initParams}',
+                {},
+                function(response) {
+
+                    window.marketplaceName = response.data.marketplaceName;
+                    window.marketplaceNetworkUrl = response.data.marketplaceNetworkUrl;
+                    window.marketplaceLogo = response.data.marketplaceLogo;
+                    window.defaultMarketplace = response.data.defaultMarketplace;
+                    window.isFixedPriceAllowed = response.data.isFixedPriceAllowed;
+                    window.purchasePriceInDetail = response.data.purchasePriceInDetail;
+
+                    me.launchAction();
+                    me.setEventListeners();
+                }
+            );
+        } else {
+            me.launchAction();
+            me.setEventListeners();
+        }
 
+        me.callParent(arguments);
```

Comment:
Since Ajax request may be executed after ``me.callParent(arguments)`` is it ok if that happens